Language/Functions/Communication/SPI.adoc

```diff
@@ -27,6 +27,22 @@ To read more about Arduino and SPI, you can visit the https://docs.arduino.cc/le
 --
 // OVERVIEW SECTION ENDS
 
+// HOW TO USE SECTION STARTS
+[#howtouse]
+--
+|================================================================================================================================================
+|                                                    Boards                                            | SPI Pins                            | SPI Headers |
+| Leonardo, UNO R3, UNO R3 SMD, UNO WiFi Rev2, UNO Mini Ltd, UNO R4 Minima, UNO R4 WiFi, Yún Rev2, Zero| 10(CS), 11(COPI), 12(CIPO), 13(SCK) | |
```

Comment:
```suggestion
| UNO R3, UNO R3 SMD, UNO WiFi Rev2, UNO Mini Ltd| 10(CS), 11(COPI), 12(CIPO), 13(SCK) | |
| UNO R4 Minima, UNO R4 WiFi| 10(CS), 11(COPI), 12(CIPO), 13(SCK) | |
| Leonardo, Yún Rev2, Zero| 10(CS), 11(COPI), 12(CIPO), 13(SCK) | |
```
Split them up a bit because too many boards in the same column (rendering was off)